spec/views/hq/users/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "hq/users/_form.html.erb", type: :view do
+
+  before :each do
+    @user = FactoryGirl.build_stubbed(:user, user_name: 'Frederick & Bloggs', email: 'fred.bloggs@example.com')
+    render
+  end
+
+  it 'has a form element' do
+    expect(rendered).to have_selector('form')
+  end
+
+  it 'has an input for user name, prefilled with current value' do
+    expect(rendered).to have_selector("input\#user_user_name[value=\"#{@user.user_name}\"]")
+  end
+
+  it 'has an input for email address, prefilled with current value' do
+    expect(rendered).to have_selector("input\#user_email[value=\"#{@user.email}\"]")
+  end
+
+  it 'has a submit button' do
+    expect(rendered).to have_selector('input[type="submit"]')
```

Comment:
Well I did take inspiration from the spec for the equivalent form in the Partners area, which I think is testing the same thing. Except that it uses assert_select and I felt I didn't fully understand what way it was working.